STM32Cube/Tasks/can_handler.c

```diff
@@ -15,10 +16,14 @@ void can_handle_rx(const can_msg_t *message, uint32_t timestamp){
 	{
 		HAL_GPIO_WritePin(GPIOB, GPIO_PIN_5, 1);
 	}
-	else if(msgtype == MSG_ACTUATOR_CMD && get_actuator_id(message) == ACTUATOR_INJECTOR_VALVE && get_curr_actuator_state(message) == ACTUATOR_ON)
+	else if(msgtype == MSG_ACTUATOR_CMD && get_actuator_id(message) == ACTUATOR_INJECTOR_VALVE && get_req_actuator_state(message) == ACTUATOR_ON)
 	{
-		xEventGroupSetBits(flightPhaseEventsHandle, INJ_OPEN_BIT); //
+		result = xEventGroupSetBitsFromISR(flightPhaseEventsHandle, INJ_OPEN_BIT, &xHigherPriorityTaskWoken);
```

Comment:
ISR safe version is based, I assume this function is called during an interrupt (sry idk can handler)?